packages/strapi-plugin-i18n/admin/src/containers/SettingsPage/LocaleSettingsPage.js

```diff
@@ -15,11 +15,12 @@ const LocaleSettingsPage = ({
   canUpdateLocale,
 }) => {
   const { formatMessage } = useIntl();
+  const [isOpenedCreateModal, setIsOpenedCreateModal] = useState(false);
 
   const actions = [
     {
       label: formatMessage({ id: getTrad('Settings.list.actions.add') }),
-      onClick: () => console.log('add locale'),
+      onClick: () => setIsOpenedCreateModal(true),
```

Comment:
You can create a toggle function here instead;

`const handleToggleModalCreate = () => {
   if (canCreate) {
     setIsOpenedCreateModal(prev=> !prev);
  }
}`

Then you'll be able to make the following change `onClick={handleToggleModalCreate}`

Reply:
I think the `canCreate` check is not necessary since the component is hidden when `canCreate` is false, and then, the click is not available 😊 

Concerning the toggle, yes, let's do that 😊